pkg/applications/applications.go

```diff
@@ -86,7 +86,7 @@ func (d Deployment) URL(kc kubernetes.Interface, a Application) string {
 	return url
 }
 
-// GetApplications fetches all Applications
+// GetApplications fetches all Apps
```

Comment:
So `GetApplication` so far where the user's "applications" right? Now you want to call this "Apps" and unify this with the current Apps plugins and all become apps? We really are in a bit of a naming mess here.

Reply:
see https://github.com/jenkins-x/enhancements/tree/master/proposals/1 but I'm also hoping to submit a further enhancement to address the mess of `jx get app` v `jx get applications` but that would be another enhancement + PR etc...